test/dialect/postgresql/test_reflection.py

```diff
@@ -1524,31 +1525,55 @@ def test_reflect_unique_index(self):
     def test_reflect_check_constraint(self):
         meta = self.metadata
 
-        cc_table = Table(
-            "pgsql_cc",
-            meta,
-            Column("a", Integer()),
-            CheckConstraint("a > 1 AND a < 5", name="cc1"),
-            CheckConstraint("a = 1 OR (a > 2 AND a < 5)", name="cc2"),
-        )
+        sql = text("""\
+            CREATE OR REPLACE FUNCTION is_positive(
+                x integer DEFAULT '-1'::integer)
+                RETURNS boolean
+                LANGUAGE 'plpgsql'
+                COST 100
+                VOLATILE 
+            AS $BODY$BEGIN
+                RETURN x > 0;
+            END;$BODY$;
+        """)
+        testing.db.execute(sql)
 
-        cc_table.create()
+        try:
+            cc_table = Table(
+                "pgsql_cc",
+                meta,
+                Column("a", Integer()),
+                CheckConstraint("a > 1 AND a < 5", name="cc1"),
+                CheckConstraint("a = 1 OR (a > 2 AND a < 5)", name="cc2"),
+                CheckConstraint("is_positive(a)", name="cc3"),
+            )
 
-        reflected = Table("pgsql_cc", MetaData(testing.db), autoload=True)
+            cc_table.create()
 
-        check_constraints = dict(
-            (uc.name, uc.sqltext.text)
-            for uc in reflected.constraints
-            if isinstance(uc, CheckConstraint)
-        )
+            reflected = Table("pgsql_cc", MetaData(testing.db),
```

Comment:
the newer syntax here is:

    reflected = Table("pgsql_cc", MetaData(), autoload_with=testing.db)